src/main/java/uk/gov/ida/verifyserviceprovider/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -38,6 +41,13 @@ public EuropeanIdentityConfiguration(@JsonProperty("hubConnectorEntityId") Strin
         this.enabled = enabled;
         this.hubConnectorEntityId = hubConnectorEntityId;
         this.trustStoreConfiguration = trustStore;
+
+        this.acceptableHubConnectorEntityIds =
+            ArrayUtils.isEmpty(acceptableHubConnectorEntityIds)
+                ? new String[] { hubConnectorEntityId }
+                : ArrayUtils.contains(acceptableHubConnectorEntityIds, hubConnectorEntityId)
+                    ? acceptableHubConnectorEntityIds
+                    : (String[])ArrayUtils.add(acceptableHubConnectorEntityIds, hubConnectorEntityId);
```

Comment:
Would be better to do `unique set` or `unique list` or something like that. You can also use lambda nicely here along the lines of: create new list containing `hubConnectorEntityId `, then add `acceptableHubConnectorEntityIds ` and make a unique set 

Also we don't use `ArrayUtils` - this is a 3rd party library. Can we use `Arrays.asList` or `Arrrays.stream` instead so we stick to using Java built-ins?

Also it would help if `acceptableHubConnectorEntityIds` was a list - using an array is clunky